app/steps/ui/payment/breakdown/index.js

```diff
@@ -21,7 +21,7 @@ class PaymentBreakdown extends Step {
 
         ctx.copies = this.createCopiesLayout(formdata);
         ctx.applicationFee = fees.applicationfee;
-        ctx.total = Number.isInteger(fees.total) ? fees.total : parseFloat(fees.total).toFixed(2);
+        ctx.total = Number.isInteger(fees.total) ? fees.total.toFixed(2) : parseFloat(fees.total).toFixed(2);
```

Comment:
This is the same as `ctx.total = parseFloat(fees.total).toFixed(2);`

Reply:
Also, after `ctx.copies = this.createCopiesLayout(formdata);` on line 22, can you also add:
```
ctx.copies.uk.cost = ctx.copies.uk.cost.toFixed(2);
ctx.copies.overseas.cost = ctx.copies.overseas.cost.toFixed(2);
```
and then remove the `.toFixed(2)` from the template?

Reply:
And one last thing, on the template, can you add `format: "numeric"` to each table row containing numbers?